cmd/create/machinepool/nodepool.go

```diff
@@ -472,13 +472,11 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		maxSurge := args.maxSurge
 		if interactive.Enabled() {
 			maxSurge, err = interactive.GetString(interactive.Input{
-				Question: "Max surge",
-				Help:     cmd.Flags().Lookup("max-surge").Usage,
-				Default:  maxSurge,
-				Required: false,
-				Validators: []interactive.Validator{
-					machinepools.ValidateUpgradeMaxSurgeUnavailable,
-				},
+				Question:   "Max surge",
+				Help:       cmd.Flags().Lookup("max-surge").Usage,
+				Default:    maxSurge,
+				Required:   false,
+				Validators: []interactive.Validator{},
```

Comment:
just want to confirm: ValidateUpgradeMaxSurgeUnavailable is not required?

Reply:
I think i removed by mistake, re-adding, thanks marco

Reply:
Ah, i think this was from a rebase, as it's not related to this MR, and I must have kept the wrong version. Either way fixed now :)